src/FileFormats/Sherlock.jl

```diff
@@ -68,7 +68,7 @@ function _read_Sherlock_POLAR(filename::String, read_activations, layer_type, re
             layers[i] = DenseLayerOp(W, b, activations(i))
         end
 
-        read_end(io)
+        return read_end(io)
```

Comment:
This is incorrect. I will file a bug report.

Reply:
Well, actually this _is_ correct. I learned something today.
Anyway, I do not like this style, so I asked in `JuliaFormatter` whether this could become an option.